src/EventSubscriber/EzcontentJsonapiSubscriber.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Drupal\ezcontent_api\EventSubscriber;
+
+use Drupal\Core\Cache\CacheableResponseInterface;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * Class EzcontentJsonapiSubscriber
+ * @package Drupal\ezcontent_api\EventSubscriber
+ */
+class EzcontentJsonapiSubscriber implements EventSubscriberInterface {
+
+  /**
+   * The config factory.
+   *
+   * @var \Drupal\Core\Config\ConfigFactoryInterface
+   */
+  protected $configFactory;
+
+  /**
+   * Constructs a EzcontentJsonapiSubscriber object.
+   *
+   * @param \Drupal\Core\Config\ConfigFactoryInterface $config_factory
+   *   The config factory.
+   */
+  public function __construct(ConfigFactoryInterface $config_factory) {
```

Comment:
We can use camelCase for all variable name, rest looks good.

Reply:
Hey @arshadkhan53 Not sure where and why we need camelCase, this is in sync with core format. Please share an example.